### DISCUSSION TITLE
persistent "trained/merged" filter

### DISCUSSION THREAD

**Discussion by FlashlightET** [civitai/civitai#205] [Ideas]
sorry if this is the wrong place, but i noticed if i set my filter to only show trained checkpoints, it resets that filter once i refresh or go to any page. unlike the other filters which remember the setting.
![image](https://user-images.githubusercontent.com/29938499/215640110-26df17fb-f45b-49b7-8403-c481ae3fd754.png)

Also unrelated, and i get this is literally a feature added *today*, but i noticed that i cant edit my old loras to mention theyre trained, thus they dont show up with the train/merge filter enabled

**Comment by JustMaier** [comment]
Thanks for the heads up on this. Looks like we forgot to get the value for that filter stored in the cookies. We'll get that fixed up this week 👍

**Comment by manuelurenah** [comment]
Hey @FlashlightET! We have already implemented a fix for this and should be out in the next release later today. Thanks again for pointing this out 💪
